Separate popover's light dismiss from Esc button close request (#30890)
mdn/content: files/en-us/web/api/popover_api/using/index.md

When a popover element is set with `popover` or `popover="auto"` as shown above, it is said to have **auto state**. The two important behaviors to note about auto state are:

- The popover can be "light dismissed" — this means that you can hide the popover by clicking outside it.

> - The popover can be closed by one of close requests, such as pressing the <kbd>Esc</kbd> key.

Reviewer: ```suggestion
- The popover can be closed using a "close request", such as pressing the <kbd>Esc</kbd> key.
```

> - The popover can be closed using browser-specific mechanisms, such as pressing the <kbd>Esc</kbd> key.

Reviewer: ```suggestion
- The popover can also be closed, using browser-specific mechanisms such as pressing the <kbd>Esc</kbd> key.
```
